### DISCUSSION TITLE
telegrram points

### DISCUSSION THREAD

**Discussion by CFarketmez** [IBAX-io/go-ibax#249] [Bug Submission]
guys hi. i have a question? i already have been joined your telegram groups how can i claim this points? thanks have nice days team.

**Comment by byblckd** [comment]
They said that they will do the ibax points for Telegram, Discord and Youtube before the airdrop. Just check the web page. Soon

**Comment by CFarketmez** [comment]
i checked before and that soon mark is just for youtube i think
